### ISSUE TITLE
Change the way to launch the web-app (or update the doc)

### ISSUE THREAD

**Issue by Philippe-Neveux** [dssg/aequitas#111]
Hello the community !

I'm using aequitas for the first time today and i was facing with some issues for launching the audit web-app locally.

To make the command `python -m serve` work, I had to clone the repository and add the following shell environment variable in my terminal `HOST=localhost` because it used in the server.py script. 

I don't know if it was the best way to do it but finally it worked. So i was wondering if it would be possible to update the documentation or to simplify the process for the audit web-app

Thanks !

**Comment by jesteria** [comment]
Sure. Are you just looking for a way to invoke that script from the `pip install …` version of the package then?

Like…?

```sh
$ pip install aequitas
…

$ aequitas-serve
```

---

As for the HOST variable you can always:

```sh
$ HOST=localhost serve.py
```

But sure this could also be a proper command argument :smile:

**Comment by Philippe-Neveux** [comment]
Yes exactly ! And maybe add a default value for the host set to localhost 😃

**Comment by jesteria** [comment]
Sure. Sounds like an enhancement to me.

**Comment by jesteria** [comment]
## Technical discussion

1. A CLI command could be added to the Python distribution, no- or little-different from the existing `aequitas-report`.
   * The existing `serve.py` could be included in the distribution via the setup.py `scripts` parameter [[docs]](https://python-packaging.readthedocs.io/en/latest/command-line-scripts.html).
   * …And/or a Python package function could be added to the `aequitas_webapp` package for CLI command invocation (via the `console_scripts` parameter) and optionally via straight Python.
   * …And/or the `aequitas_webapp` package could be given a `__main__.py` such that it is invokable by `python -m aequitas_webapp` (without downloading the source). **Dealer's choice?**
2. The `serve.py` script and/or derivative can read the environment and/or an argument. There's `argparse`. There's `sys.argv`. _Etc._
    ```python
    host = sys.argv[1] if len(sys.argv) > 1 else os.getenv('HOST')
    ```
